packages/devtools-reps/src/object-inspector/tests/component/create-long-string-client.js

```diff
@@ -58,22 +55,19 @@ describe("createLongStringClient", () => {
             }
           ],
           createObjectClient: ObjectClient(stub),
-          createLongStringClient
+          createLongStringClient: grip => LongStringClient(grip, { substring })
```

Comment:
Omitted LongStringClient spy and assertion since it's checked on a previous test.